code/modules/vehicles/vehicle_actions.dm

```diff
@@ -336,6 +336,8 @@
 /datum/action/vehicle/sealed/noise
 	var/sound_path = 'sound/items/carhorn.ogg'
 	var/sound_message = "makes a sound."
+	///Signal fired by the action, whenever it's triggered. Used by the Vim circuit component.
+	var/signal_fired
```

Comment:
I would prefer each signal was sent from an overrided trigger, rather then this.
I know it could be deduplicated, and normally that'd be great, but I value signal sends being unique semi highly

Reply:
aight, i'll take care of that

Reply:
done